ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/_cq_dialog/.content.xml

```diff
@@ -39,13 +40,13 @@
                                     wrapperClass="cmp-adaptiveform-textinput__value"
                                     jcr:primaryType="nt:unstructured"
                                     sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
-                                    fieldLabel="Default Value"
+                                    fieldLabel="Default text"
```

Comment:
Default value would be consistent across dialogs - text conflicts with Label text and more generic